monitor/utils/events.js

```diff
@@ -16,12 +16,14 @@ const {
 } = require('../../commons')
 const { normalizeEventInformation } = require('./message')
 const { filterTransferBeforeES } = require('./tokenUtils')
+const { writeFile, readCacheFile } = require('./file')
 
 const {
   COMMON_HOME_RPC_URL,
   COMMON_FOREIGN_RPC_URL,
   COMMON_HOME_BRIDGE_ADDRESS,
-  COMMON_FOREIGN_BRIDGE_ADDRESS
+  COMMON_FOREIGN_BRIDGE_ADDRESS,
+  MONITOR_CACHE_EVENTS
```

Comment:
Will it still work if the variable is not defined at all?

Reply:
Yes, it will. In this case `MONITOR_CACHE_EVENTS` will be equal to `undefined`.